schematic/store/synapse.py

```diff
@@ -557,25 +558,36 @@ def getFilesInStorageDataset(
         file_list = []
 
         # iterate over all results
-        for dirpath, dirname, filenames in walked_path:
+        for dirpath, _, path_filenames in walked_path:
             # iterate over all files in a folder
-            for filename in filenames:
-                if (not "manifest" in filename[0] and not fileNames) or (
-                    fileNames and filename[0] in fileNames
+            for path_filename in path_filenames:
+                if ("manifest" not in path_filename[0] and not fileNames) or (
+                    fileNames and path_filename[0] in fileNames
```

Comment:
Just making sure I have this right, this is to make sure `fileNames is not None` then it checks `path_filename[0] in fileNames`

Reply:
There are 2 things occurring here, either:

1. We include all files that were walked (Except for any files that contain the word "manifest"
2. We include only those files in the `fileNames` list